scripts/enter-env.sh

```diff
@@ -16,7 +22,7 @@ TOP_DIR="$(realpath "${SETUP_DIR}/..")"
 if [ $SOURCED = 0 ]; then
 	echo "You must source this script, rather than try and run it."
 	echo ". $SETUP_SRC"
-	exit 1
+	return 1
```

Comment:
This is triggered if the script is run (instead of sourced) so it should be an `exit 1`?

Reply:
This should probably be the first check then too, I guess?

Reply:
True, I was too hasty. The sourcing check is bash specific though...